### PR TITLE
Added doc for synthetic events

### PR DESCRIPTION
**Pull request by kofifus**
Added a brief description of invoking actions in response to synthetic events

### CODE REVIEW

File: docs/api/h.md [hunk 1]
```diff
@@ -169,6 +169,8 @@ const pokedex = (pokemon) =>
 
 Props that represent event listeners, such as [`onclick`](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event), [`onchange`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event), [`oninput`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event), etc. are where you would assign [actions](../architecture/actions.md) to VNodes.
 
+Synthetic events can be added in the same way as long as their name starts with "on", so an event created with `const buildEvent = new Event('build')` can be used like this: `h("button", { onbuild: BuildAction }, text("Click Me"))`
```
Comment (jorgebucaran) [inline docs/api/h.md:172]: Can you use full code blocks? Thanks.

**Review by jorgebucaran — CHANGES_REQUESTED** [review]

File: docs/api/h.md (hunk 1 above)
Comment (kofifus) [reply docs/api/h.md]: done

File: docs/api/h.md [hunk 2]
```diff
@@ -169,6 +169,15 @@ const pokedex = (pokemon) =>
 
 Props that represent event listeners, such as [`onclick`](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event), [`onchange`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event), [`oninput`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event), etc. are where you would assign [actions](../architecture/actions.md) to VNodes.
 
+Synthetic events can be added in the same way as long as their name starts with "on", so an event created with 
+```js
+const buildEvent = new Event('build')
```
Comment (jorgebucaran) [inline docs/api/h.md:174]: Double quotes for consistency, thanks.

**Review by jorgebucaran — CHANGES_REQUESTED** [review]

File: docs/api/h.md (hunk 2 above)
Comment (kofifus) [reply docs/api/h.md:174]: Sorry I don't understand, in the event name ? "build" ?

Comment (kofifus) [reply docs/api/h.md]: done

**Comment by jorgebucaran** [comment]
Awesome, thanks! 🙏

**Pull request merged**